### PR TITLE
Feature: Add Open File Shortcut

### PR DESCRIPTION
**Pull request by Awesomeplayer165**
Command + O and Control + O.

### CODE REVIEW

File: src/document/DocumentManager.ts [hunk 1]
```diff
@@ -245,6 +245,9 @@ export class DocumentManager {
       this.model.uiState.setSelectedSidebarItem(undefined);
       this.model.uiState.setSelectedNavbarItem(-1);
     });
+    hotkeys("ctrl+o,command+o", () => {
+      this.saveFileDialog();
```
Comment (calcmogul) [inline src/document/DocumentManager.ts:249]: This opens the save file dialog, not the open file dialog.

Comment (calcmogul) [reply src/document/DocumentManager.ts:249]:
```suggestion
      invoke("open_file_dialog");
```

Comment (Awesomeplayer165) [reply src/document/DocumentManager.ts:249]: yeah not sure if it is possible with the current setup. ill need to refactor things around from mobx context to documentManager.

File: src/document/DocumentManager.ts [hunk 2]
```diff
@@ -242,9 +242,20 @@ export class DocumentManager {
     });
     hotkeys.unbind();
     hotkeys("escape", () => {
+      this.model;
```
Comment (calcmogul) [inline src/document/DocumentManager.ts:245]: This line seems unnecessary.

**Review by calcmogul — CHANGES_REQUESTED** [review]

File: src/document/DocumentManager.ts (hunk 2 above)
Comment (Awesomeplayer165) [reply src/document/DocumentManager.ts:245]: not sure why that was even added..

Comment (Awesomeplayer165) [reply src/document/DocumentManager.ts:245]: I don't see this on my branch actually..

**Review by calcmogul — APPROVED** [review]

**Pull request merged**